code/processes/housekeeping.q

```diff
@@ -57,7 +57,7 @@ csvloader:{[CSV]
 //-Sees if the function in the CSV file is in the function list. if so- it carries out that function on files that match the parameters in the csv [using find function]
 wrapper:{[DICT]
 	$[not DICT[`function] in key `.;.lg.e[`housekeeping;"Could not find function: ",string DICT[`function]];
-	(value DICT[`function]) each (find[.rmvr.removeenvvar [DICT[`path]];DICT[`match];DICT[`age];DICT[`agemin]] except find[.rmvr.removeenvvar [DICT[`path]];DICT[`exclude];DICT[`age];DICT[`agemin]])]}
+	(value DICT[`function]) each (find[.rmvr.removeenvvar [DICT[`path]];DICT[`match];DICT[`age];DICT[`agemin];DICT[`checkfordirectory]] except find[.rmvr.removeenvvar [DICT[`path]];DICT[`exclude];DICT[`age];DICT[`agemin];DICT[`checkfordirectory]])]}
```

Comment:
Last line of this function is a bit hard to read and features very similar uses of find, by building a projection `p` and passing the match/exclude terms separately this can be cleaned up nicely:
```
wrapper:{[DICT]
    if[not DICT[`function]in key`.;:.lg.e[`housekeeping;"Could not find function: ",string DICT`function]];
    p:find[.rmvr.removeenvvar DICT`path;;DICT`age;DICT`agemin];
    value[DICT`function]each p[DICT`match]except p DICT`exclude}
```